app/posts/govuk-prototype-filters.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Rapidly prototype with data using a common set of template filters
+description: Introducing a collection of Nunjucks template filters to help manipulate and transform data used in prototypes.
+date: 2023-05-21
+tags: post
+layout: post
+author:
+  name: Paul Lloyd
+  url: https://github.com/paulrobertlloyd
+image:
+  src: /assets/posts/govuk-prototype-filters/govuk-prototype-filters.png
+  alt: The homepage illustration for the GOV.UK Prototype Filters.
```

Comment:
Think the alt text should describe the image? Or else maybe it’d be better for it to be blank (empty string) if it’s purely decorative? 🤔 

```suggestion
  alt: Illustration titled 'GOV.UK Prototype Filters' showing checkboxes and radio buttons being generated from code.
```